app/human-detector-app/App.tsx

```diff
@@ -2,30 +2,75 @@ import * as React from 'react';
 import { View, Text, StyleSheet, Platform } from 'react-native';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import { TokenResponse } from 'expo-auth-session';
+import JWT from 'expo-jwt';
+import * as CryptoJS from 'crypto-js';
+import User from './classes/User';
 import CameraScreen from './screens/CameraScreen';
 import GroupScreen from './screens/GroupScreen';
 import LoginScreen from './screens/LoginScreen';
+import { getExponentPushToken, sendExpoNotifToken } from './src/notifications/notifTokenInit';
 
 const Stack = createNativeStackNavigator();
 
 export default function App(): React.ReactElement {
+  let isUserSignedIn = false;
+
+  // Update upon getting a token
+  const [tokenResponse, setTokenResponse] = React.useState(null);
+
+  // Check if the token is expired
+  if (tokenResponse != null) {
+    // TODO: refresh token if expired
```

Comment:
As mentioned in the description, this area will still need work.  I just added it here as todos for now and included the notification flow that this task covers.